skaffold/src/test/kotlin/com/google/container/tools/skaffold/SkaffoldFilesTest.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.skaffold
+
+import com.google.common.truth.Truth.assertThat
+import com.google.container.tools.test.ContainerToolsRule
+import com.intellij.mock.MockVirtualFile
+import org.junit.Rule
+import org.junit.Test
+
+/** Unit tests for Skaffold files functionality in [SkaffoldFiles.kt] */
+class SkaffoldFilesTest {
+    @get:Rule
+    val containerToolsRule = ContainerToolsRule()
+
+    @Test
+    fun `valid skaffold file is accepted`() {
+        val skaffoldFile = MockVirtualFile.file("skaffold.yaml")
+        skaffoldFile.setText("apiVersion: skaffold/v1alpha2")
+
+        assertThat(isSkaffoldFile(skaffoldFile)).isTrue()
+    }
+
+    @Test
+    fun `frivolously formatted skaffold file is accepted`() {
```

Comment:
maybe be more specific than "frivolously"? like something about whitespace if thats what you are testing.

Reply:
sounds good, changed to a more specific name.